data/json/items/gun/shot.json

```diff
@@ -653,7 +653,7 @@
     "durability": 6,
     "clip_size": 12,
     "reload": 220,
-    "barrel_volume": "360 ml",
+    "barrel_volume": "505 ml",
```

Comment:
I updated it to represent a more precise measurement based on the 18.5 inch (470 mm) barrel version.

π or 3.14 x 1.85 x 1.85 = 10.74665 x 47 = 505.09255, rounded to 505 cm3 because 1ml = 1cm3. 1 litre = 1,000 ml = 1,000 cm3; the result is 505 ml.